src/components/dropdown/dropdown.js

```diff
@@ -62,7 +62,7 @@ export default {
         class: t.menuClasses,
         attrs: {
           role: t.role,
-          'aria-labelledby': t.safeId(split ? '_BV_toggle_' : '_BV_button_')
+          'aria-labelledby': t.safeId(this.split ? '_BV_toggle_' : '_BV_button_')
```

Comment:
This should be `t.split`, and not `this.split` for minification reasons.

Reply:
fixed in #1500